src/utility/STM32Cube_FW/tl.h

```diff
@@ -0,0 +1,314 @@
+/**
+ ******************************************************************************
+ * @file    tl.h
```

Comment:
same here : here did you get tl.h from ?
there are many applications, probably the one we need is from transparent mode

Reply:
close to zephyr tl.h and mostly coming from ./STM32CubeWB/Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl